intranet/templates/error/static/503.html

```diff
@@ -83,10 +83,10 @@
         <div class="center">
             <div class="error">
                 <div class="logo"></div>
-                <h1>Internal Server Error - HTTP 503</h1>
+                <h1>Intranet is temporarily unavailable</h1>
```

Comment:
```suggestion
                <h1>Intranet is temporarily unavailable - HTTP 503</h1>
```
For easier debugging from screenshots